neoload/commands/project.py

```diff
@@ -7,17 +7,24 @@
 @click.argument("command", required=True, type=click.Choice(['up', 'upload', 'meta']))
 @click.option("--path", "-p", type=click.Path(exists=True), default='.',
               help="path of project folder, zip or yml file. . is default value")
+@click.option("--display-progress/--just-json", required=False, default=True,
+              help=("Suppress the real-time progress when files is above " + str(neoLoad_project.MAX_FILE_MB_BEFORE_PROGRESS) + "MB") )
 @click.argument("name_or_id", type=str, required=False)
-def cli(command, name_or_id, path):
+def cli(command, name_or_id, path, display_progress):
     """Upload and list scenario from settings"""
     if not name_or_id or name_or_id == "cur":
         name_or_id = user_data.get_meta(test_settings.meta_key)
 
     if not tools.is_id(name_or_id):
         name_or_id = test_settings.__resolver.resolve_name(name_or_id)
 
+    suppress_progress = not display_progress
+
+    if display_progress and not tools.is_user_interactive():
```

Comment:
Why suppress_progress is set to True when display_progress is True ?? The meaning of word is important if we want maintenability.

Reply:
Resolved by discussion today to use CI-specific indicators instead of graphics